karma.mc.config.js

```diff
@@ -51,11 +51,18 @@ module.exports = function(config) {
       resolveLoader: {alias: {inject: path.join(__dirname, "loaders/inject-loader")}},
       // This resolve config allows us to import with paths relative to the system-addon/ directory, e.g. "lib/ActivityStream.jsm"
       resolve: {
+        extensions: [".js", ".jsx"],
         modules: [
           PATHS.systemAddonDirectory,
           "node_modules"
         ]
       },
+      externals: {
+        // enzyme needs these for some reason
```

Comment:
It's a backwards compatibility reason.  I'd suggest linking to <https://github.com/airbnb/enzyme/blob/master/docs/guides/webpack.md#using-enzyme-with-webpack>, because at some point, they will presumably drop the React 0.13 compat, and change this document to match.